src/Umbraco.Web.BackOffice/Filters/DataTypeValidateAttribute.cs

```diff
@@ -98,13 +95,13 @@ public override void OnActionExecuting(HttpActionContext actionContext)
                 // run each IValueValidator (with null valueType and dataTypeConfiguration: not relevant here)
                 foreach (var validator in editorField.Validators)
                 foreach (var result in validator.Validate(field.Value, null, null))
-                    actionContext.ModelState.AddValidationError(result, "Properties", field.Key);
+                    context.ModelState.AddModelError(field.Key,result.ErrorMessage);
```

Comment:
this isn't right, we cannot remove "Properties" here, the validation keys returned are very specific, any reason why this isn't just still using AddValidationError?

Reply:
My bad.. Did not realize this was an extention method. I migrated this extension method.